app/build.gradle

```diff
@@ -6,12 +6,15 @@ android {
 
     defaultConfig {
         applicationId "com.compscieddy.meetinthemiddle"
-        minSdkVersion 17
+        minSdkVersion 19
         targetSdkVersion 23
         versionCode 1
         versionName "1.0"
         multiDexEnabled true
     }
+    dexOptions {
+        javaMaxHeapSize "4g"
```

Comment:
Ooh can you explain these options more? maybe add a comment. The gradle file is still very much a mystery to me so I'm always interested in hearing more about the potential options